packages/firestore/package.json

```diff
@@ -46,7 +44,8 @@
     "test:node:persistence:prod": "node ./scripts/run-tests.js --main=index.node.ts --persistence 'test/{,!(browser|lite)/**/}*.test.ts'",
     "test:travis": "ts-node --compiler-options='{\"module\":\"commonjs\"}' ../../scripts/emulator-testing/firestore-test-runner.ts",
     "test:minified": "(cd ../../integration/firestore ; yarn test)",
-    "api-report": "rm -rf temp && api-extractor run --local --verbose",
+    "prepare": "yarn build:release",
+    "api-report": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' ts-node ./scripts/api-report.js",
```

Comment:
Should we execute `api-report.ts` directly since we are using ts-node. Then we won't need to commit the compiled js.

Reply:
The compiled JS gets committed because of the `yarn build:scripts` rule which is needed because of `rollup`. I fixed this line though since using `ts-node` and the `.js` script is pretty... awkward.